CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing
+
+When contributing for the first time, try to find an issue labeled with
+"good first issue", and leave a comment stating that you would like to work on
```

Comment:
Why not directly link it? https://github.com/FAForever/server/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22

Reply:
Didn't think about it. Good idea